<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineSettings</class>
 <widget class="QDialog" name="MachineSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Machine Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_Processor">
     <property name="title">
      <string>Processor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Processor(s):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_processors">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Execution Cap:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_executionCap">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>(1% - 100%)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Extended Features:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBox_enablePAEnx">
        <property name="text">
         <string>Enable PAE/NX</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>(1 CPU - 4 CPUs)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_Acceleration">
     <property name="title">
      <string>Acceleration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Paravirtualization Interface:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_paravirtInterface">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Legacy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minimal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hyper-V</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KVM</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Hardware Virtualization:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_enableVTxAMDV">
        <property name="text">
         <string>Enable VT-x/AMD-V</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_enableNestedPaging">
        <property name="text">
         <string>Enable Nested Paging</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="OverlayButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Motherboard">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Motherboard</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="1" colspan="2">
       <widget class="QCheckBox" name="extFeatCheckBox_enableEFI">
        <property name="text">
         <string>Enable EFI (special OSes only)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Boot Order</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Base Memory</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pointing Device</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_chipset">
        <item>
         <property name="text">
          <string>PIIX3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ICH9</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>(4MB - 8192 MB)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QCheckBox" name="extFeatCheckbox_enableIO">
        <property name="text">
         <string>Enable I/O APIC</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Extended Features</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_pointingDevice">
        <item>
         <property name="text">
          <string>PS/2 Mouse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USB Tablet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USB Multi-Touch Tablet</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QCheckBox" name="bootOrderCheckbox_network">
        <property name="text">
         <string>Network</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="bootOrderCheckbox_optical">
        <property name="text">
         <string>Optical</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinbox_baseMemory">
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Chipset</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QCheckBox" name="extFeatCheckbox_hwClockUTCtime">
        <property name="text">
         <string>Hardware Clock in UTC Time</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="bootOrderCheckBox_floppy">
        <property name="text">
         <string>Floppy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="bootOrderCheckbox_hardDisk">
        <property name="text">
         <string>Hard Disk</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OverlayButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>MachineSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OverlayButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>MachineSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
